pom.xml

```diff
@@ -448,15 +448,13 @@ Import-Package: \\
             <strictCheck>true</strictCheck>
             <aggregate>true</aggregate>
             <mapping>
-              <groovy>JAVADOC_STYLE</groovy>
-              <java>JAVADOC_STYLE</java>
-              <mwe2>JAVADOC_STYLE</mwe2>
-              <xtend>JAVADOC_STYLE</xtend>
+              <mwe2>SLASHSTAR_STYLE</mwe2>
+              <xtend>SLASHSTAR_STYLE</xtend>
```

Comment:
Is this change intended?

Reply:
Yes see https://github.com/openhab/openhab-addons/pull/18015#issuecomment-2567034802.
I'll split it up as SAT needs to be updated.
A new SAT release will also help with being able to analyze Java 21 constructs because of the dependency upgrades.